### DISCUSSION TITLE
How to customize the figure name?

### DISCUSSION THREAD

**Discussion by drriguz** [quarto-dev/quarto-cli#4430] [Q&A]
Hi,

If we reference a figure, it will be displayed as "Figure 1.1 Some awesome picture",
for users writing non-English articles they may want to change it to "图 1.1" or something else,
is there anyway to define a mapping between the figure type and it's display name?

**Comment by mcanouil** [comment]
See the lang option <https://quarto.org/docs/authoring/language.html#custom-translations>.
The language key is `crossref-fig-title`.

**Reply by drriguz** [reply]
Thanks! I'll check it.
